### PR TITLE
slither-analyzer 0.9.6

### PR DESCRIPTION
**Pull request by chenrui333**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [ ] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [ ] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [ ] Does your build pass `brew audit --strict <formula>` (after doing `brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

-----

### CODE REVIEW

File: Formula/slither-analyzer.rb [hunk 1]
```diff
@@ -148,18 +153,13 @@ class SlitherAnalyzer < Formula
   end
 
   resource "protobuf" do
-    url "https://files.pythonhosted.org/packages/e5/9d/20e9bf4067e85c3074f1f5bac820a3cfb9ce885cddd8a649fe3570659c77/protobuf-4.23.3.tar.gz"
-    sha256 "7a92beb30600332a52cdadbedb40d33fd7c8a0d7f549c440347bc606fb3fe34b"
-  end
-
-  resource "pypandoc" do
```
Comment (elopez) [inline Formula/slither-analyzer.rb:155]: The build failure can likely be resolved by re-adding this pypandoc version

Comment (elopez) [reply Formula/slither-analyzer.rb:155]: For context, here's the upstream [issue](https://github.com/ethereum/pyrlp/issues/137) and [PR](https://github.com/ethereum/pyrlp/pull/136) on pyrlp.

Comment (chenrui333) [reply Formula/slither-analyzer.rb:155]: make sense.

Comment (elopez) [reply Formula/slither-analyzer.rb:155]: I see you readded it but with a newer version; pypandoc's `convert` (the missing function causing the build failure) was removed in 1.8, that's why the version pinned in the formula is 1.7.5. Can you roll it back to the previous version? thanks!

Comment (chenrui333) [reply Formula/slither-analyzer.rb:155]: I think upstream should also pin to use 1.7.5 as well?

Comment (elopez) [reply Formula/slither-analyzer.rb:155]: Yeah probably! But ultimately the real fix would be for `pyrlp` to discontinue using `setuptools-markdown` which has been deprecated in favor of the builtin `setuptools` functionality. (the dependency goes something like slither-analyzer -> ... -> pyrlp -> setuptools_markdown (deprecated) -> pypandoc)

**Comment by elopez** [comment]
it seems one of the new transitive dependencies requires rust to build

```
2023-07-07T13:30:26.2070660Z         running build_ext
2023-07-07T13:30:26.2070700Z         running build_rust
2023-07-07T13:30:26.2070790Z         error: can't find Rust compiler
```

The PR builds locally with this addition:

```ruby
depends_on "rust" => :build # rpds-py resource requires rust to build
```

**Review by carlocab — APPROVED** [review]

**Pull request merged**